pkg/manifests/config.go

```diff
@@ -66,6 +66,21 @@ func (c Config) IsStorageConfigured() bool {
 	return prometheusK8sConfig.VolumeClaimTemplate != nil
 }
 
+func (c Config) HasInconsistentAlertmanagerConfigurations() bool {
+	if c.ClusterMonitoringConfiguration == nil && c.UserWorkloadConfiguration == nil {
+		return false
+	}
+
+	alertmanagerMainConfig := c.ClusterMonitoringConfiguration.AlertmanagerMainConfig
+	alertmanagerUWMConfig := c.UserWorkloadConfiguration.Alertmanager
+
+	if alertmanagerMainConfig == nil && alertmanagerUWMConfig == nil {
```

Comment:
Shouldn't this be `||` instead of `&&` 
What is main is enabled by there is no uwm ? 
main == nil => false && uwm == nil => true  ==> false

now the code below uwm.Enabled will crash, won't it?

Reply:
Ahhh good catch, when I changed it I forgot to change the && to || 😅 
I've also added another test to cover that case 😉